### PR TITLE
dust: add page

### LINKED ISSUE #5047: page request: dust
I haven't seen anything in the current repo for `dust`. If it's the case, I'd like to add a page for it.

Here is the link to dust.
https://github.com/bootandy/dust

### PR DESCRIPTION
**Pull request by diohabara**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Close #5047

### CODE REVIEW

File: pages/common/dust.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# dust
+
+> Dust gives you an instant overview of which directories are using disk space:.
+> More information: <https://github.com/bootandy/dust>.
+
+- Show the current directory's information:
+
+`dust`
+
+- Show the information of A and B directories in the current directory:
```
Comment (navarroaxel) [inline pages/common/dust.md:10]:
```suggestion
- Display information for a space-separated list of directories:
```

File: pages/common/dust.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`dust A B`
```
Comment (navarroaxel) [inline pages/common/dust.md:12]:
```suggestion
`dust {{path/to/directory1}} {{path/to/directory2}}`
```

File: pages/common/dust.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show only 30 directories instead of showing default number of directories:
```
Comment (navarroaxel) [inline pages/common/dust.md:14]:
```suggestion
- Display 30 directories (defaults to 21):
```

File: pages/common/dust.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`dust -n 30`
```
Comment (navarroaxel) [inline pages/common/dust.md:16]: We prefer long argument format:
```suggestion
`dust --number-of-lines {{30}}`
```

File: pages/common/dust.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show 3 levels of subdirectories:
+
+`dust -d 3`
```
Comment (navarroaxel) [inline pages/common/dust.md:20]: The argument option should be between curly braces:
```suggestion
`dust --depth {{3}}`
```

File: pages/common/dust.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show the information of A and B directories in the current directory:
+
+`dust A B`
+
+- Show only 30 directories instead of showing default number of directories:
+
+`dust -n 30`
+
+- Show 3 levels of subdirectories:
```
Comment (navarroaxel) [inline pages/common/dust.md:18]:
```suggestion
- Display information for the current directory, up to 3 levels deep:
```

File: pages/common/dust.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`dust -d 3`
+
+- Severse order of output, with root at the lowest:
+
+`dust -r`
```
Comment (navarroaxel) [inline pages/common/dust.md:24]:
```suggestion
`dust --reverse`
```

File: pages/common/dust.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show only 30 directories instead of showing default number of directories:
+
+`dust -n 30`
+
+- Show 3 levels of subdirectories:
+
+`dust -d 3`
+
+- Severse order of output, with root at the lowest:
```
Comment (navarroaxel) [inline pages/common/dust.md:22]:
```suggestion
- Display the biggest directories at the top in descending order:
```

File: pages/common/dust.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+`dust -r`
+
+- Ignore all files and directories with the name 'ignore':
+
+`dust -X ignore`
```
Comment (navarroaxel) [inline pages/common/dust.md:28]:
```suggestion
`dust --ignore-directory {{file_or_directory_name}}`
```

File: pages/common/dust.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show 3 levels of subdirectories:
+
+`dust -d 3`
+
+- Severse order of output, with root at the lowest:
+
+`dust -r`
+
+- Ignore all files and directories with the name 'ignore':
```
Comment (navarroaxel) [inline pages/common/dust.md:26]:
```suggestion
- Ignore all files and directories with a specific name:
```

File: pages/common/dust.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+`dust -X ignore`
+
+- Do not show percentages or draw ASCII bars:
+
+`dust -b`
```
Comment (navarroaxel) [inline pages/common/dust.md:32]:
```suggestion
`dust --no-percent-bars`
```

File: pages/common/dust.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+- Severse order of output, with root at the lowest:
+
+`dust -r`
+
+- Ignore all files and directories with the name 'ignore':
+
+`dust -X ignore`
+
+- Do not show percentages or draw ASCII bars:
```
Comment (navarroaxel) [inline pages/common/dust.md:30]:
```suggestion
- Do not display percent bars and percentages:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @diohabara, welcome and thanks for your submission. I left some comments.

**Comment by diohabara** [comment]
@navarroaxel 
Should I `rebase -i` to put together these commits?

**Comment by bl-ue** [comment]
@diohabara yes, please do.  Next time you should click the "Add suggestion to batch" button on each suggestion in the "Files changed" tab.  You could have then applied all of the suggestions in one commit, to avoid this :)

See the [docs](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes).

**Comment by navarroaxel** [comment]
> Should I `rebase -i` to put together these commits?

@diohabara no, it's ok, we squash before merge.

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
Oh okay @navarroaxel.  Good to know.

**Review by bl-ue — APPROVED** [review]

**Pull request merged**